docs/content/users/quickstart.md

```diff
@@ -159,19 +159,30 @@ ddev launch
 
 ## Drupal
 
-=== "Drupal 10"
+=== "Drupal"
+    For all versions of Drupal 8+ the Composer techniques work. The settings configuration is done differently for each Drupal version, but the project type is "drupal".
 
     ```bash
-    mkdir my-drupal10-site
-    cd my-drupal10-site
-    ddev config --project-type=drupal10 --docroot=web
+    mkdir my-drupal-site
+    cd my-drupal-site
+    ddev config --project-type=drupal --php-version=8.3 --docroot=web
```

Comment:
I wouldn't make a drupal 10 instance use php 8.3. the recommended version of php for 10.x is still only 8.1.6 according to https://www.drupal.org/docs/getting-started/system-requirements/php-requirements (8.3. for example is only supported by drupal 10.2+ and i think a version should be picked that supports drupal 10.0). so i think for drupal 10 the drupal app type default php 8.2 might be the right pick  so `--php-version=8.3` might be dropped here.

php 8.3 and --corepack-enable for drupal 11 looks good. 

and going with php 8.1 for drupal 9 as well. gotta test all quickstarts tomorrow with drush another time to make sure.

and in general packing everything into a single tab and codeblock for setting up drupal 9 , 10 and 11 i consider hard to process. i think the better choice would be to go with dedicated tabs for each of the versions.

Reply:
Since Drupal 10.2 supports PHP8.3, and since that's what we're getting with composer create, I think it's OK, https://www.drupal.org/docs/getting-started/system-requirements/php-requirements

TYPO3 has been getting along with this kind of ambiguity successfully for years and many major releases, but of course they had @gilbertsoft watching out for them. 

I think your concerns are valid and we'll see what people's experience is. Separate tabs is definitely a possibility. Also, I was thinking about a `ddev config --detect` which would detect the current version and update config explicitly. That could be added to quickstarts after the initial setup.

Reply:
i am not a fan of ambiguity and making the user think. therefore are situations like that thin ice if the person using the functionality doesnt know all the details about. 

separate tabs would be a short term fix. but as i said not that elegant and it would be sort of countering the elegance and the fact that there is a `drupal`app type in place. but your idea of having a functionality of `ddev config --detect`to trigger the detection of the current version and updating the config explicitly would solve all of that in a way more elegant manner and solve the tricky situation with drupal. that way also only a single drupal tab would be necessary. and one idea would it make sense to trigger `ddev config --detect`automatically when running  `ddev composer create`? just as an additional idea. but in general big plus one for the config detect idea!